long-running/longrunning.py

```diff
@@ -65,6 +65,51 @@ def recreate_db():
     conn.commit()
 
 
+def load_logfiles():
+    conn = connect_sql()
+
+    for g in logs:
+        print("Found logs {0}".format(len(g)))
+        for path in g:
+            print("Processing: {0}".format(path))
+            path_parts = path.split('/')
+            logname = os.path.basename(path)
+
+            datestr = logname.\
+                replace('api.jujucharms.com.log-', '').\
+                replace('.anon.gz', '')
+
+            c = conn.cursor()
+            res = c.execute('''
```

Comment:
This is the new bit here. This was pulled out to be shared between init and update commands. This bit here does the check if the log file's been loaded before.